<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomersListView</class>
 <widget class="QWidget" name="CustomersListView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Customers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabEnterpriseCustomer">
      <attribute name="title">
       <string>Enterprise Customer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addEnterpriseCustomerPushButton">
            <property name="toolTip">
             <string>Add enterprise customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/add_user.png</normaloff>:/images/users/add_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editEnterpriseCustomerPushButton">
            <property name="toolTip">
             <string>Edit enterprise customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/edit_user.png</normaloff>:/images/users/edit_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeEnterpriseCustomerPushButton">
            <property name="toolTip">
             <string>Remove enterprise customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/remove_user.png</normaloff>:/images/users/remove_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>454</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTableWidget" name="enterpriseCustomerTableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>CNPJ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fantasy Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Fax Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Contact Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Email</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Street/Avenue</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Complement</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Square</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>City</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Postal Code</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPhysicalCustomer">
      <attribute name="title">
       <string>Physical Customer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addPhysicalCustomerPushButton">
            <property name="toolTip">
             <string>Add new physical customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/add_user.png</normaloff>:/images/users/add_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editPhysicalCustomerPushButton">
            <property name="toolTip">
             <string>Edit physical customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/edit_user.png</normaloff>:/images/users/edit_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removePhysicalCustomerPushButton">
            <property name="toolTip">
             <string>Remove physical customer</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../images.qrc">
              <normaloff>:/images/users/remove_user.png</normaloff>:/images/users/remove_user.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>454</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="physicalCustomerTableWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>CPF</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Birth Date</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Email</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Street/Avenue</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Complement</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Square</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>City</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>State</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Postal Code</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
